exercises/protein-translation/protein_translation_test.go

```diff
@@ -0,0 +1,63 @@
+package protein
+
+import (
+	"testing"
+)
+
+const targetTestVersion = 1
+
+type testCase struct {
+	input    string
+	expected string
+}
+
+var codonTestCases = []testCase{
+	{"AUG", "Methionine"},
+	{"UUU", "Phenylalanine"},
+	{"UUC", "Phenylalanine"},
+	{"UUA", "Leucine"},
+	{"UUG", "Leucine"},
+	{"UCU", "Serine"},
+	{"UCC", "Serine"},
+	{"UCA", "Serine"},
+	{"UCG", "Serine"},
+	{"UAU", "Tyrosine"},
+	{"UAC", "Tyrosine"},
+	{"UGU", "Cysteine"},
+	{"UGC", "Cysteine"},
+	{"UGG", "Tryptophan"},
+	{"UAA", "STOP"},
+	{"UAG", "STOP"},
+	{"UGA", "STOP"},
+}
+
+var proteinTestCases = []testCase{
+	{"AUGUUUUCUUAAAUG", "Methionine Phenylalanine Serine"},
+	{"AUGUUUUGG", "Methionine Phenylalanine Tryptophan"},
+	{"AUGUUUUAA", "Methionine Phenylalanine"},
+	{"UGGUGUUAUUAAUGGUUU", "Tryptophan Cysteine Tyrosine"},
+}
+
+func TestCodon(t *testing.T) {
+	for _, test := range codonTestCases {
+		actual := proteinLookUp(test.input)
```

Comment:
@petertseng I am OK in exporting names. Let me know any example exercise which does this ?